packages/map/src/components/Sidebar.js

```diff
@@ -114,7 +115,7 @@ const Sidebar = (props) => {
 
   return (
     <ErrorBoundary component="Sidebar">
-      <aside id="legend" className={`${legend.position} ${legend.singleColumn ? 'single-column cdcdataviz-sr-focusable' : 'cdcdataviz-sr-focusable'} ${viewport}`} role="region" aria-label="Legend" tabIndex="0">
+      <aside id="legend" className={`${legend.position} ${className} cdcdataviz-sr-focusable ${viewport}`} role="region" aria-label="Legend" tabIndex="0">
```

Comment:
@Atash3000 - good idea condensing this. You could possibly go a little further and put the classes into an array, ie

```
let columnLogic = (legend.position==='side' && legend.singleColumn) ? 'single-column' : (legend.position==='bottom' && legend.singleRow) ? 'single-row' : '';

let classNames = [
'cdcdataviz-sr-focusable',
`${viewport}`,
`${columnLogic}
]
```
and then use `classNames.join(' ')` in the class section